src/setup/steps/uninstallPackages.ts

```diff
@@ -2,5 +2,5 @@ import { $ } from "execa";
 
 export async function uninstallPackages() {
 	await $`pnpm remove chalk execa js-yaml`;
-	await $`pnpm remove @clack/prompts @octokit/request-error @types/js-yaml @types/prettier all-contributors-cli c8 globby octokit npm-user replace-in-file title-case tsx -D`;
+	await $`pnpm remove @clack/prompts @octokit/request-error @types/js-yaml @types/prettier all-contributors-cli c8 globby octokit npm-user replace-in-file title-case git-remote-origin-url git-url-parse @types/git-url-parse tsx -D`;
```

Comment:
[Style] Could you keep these alphabetized please? It can be difficult to look through these lists otherwise.

_(not a blocker for merge IMO, just a nice to have)_

Reply:
I agree.